src/core/unicode.js

```diff
@@ -1630,8 +1630,34 @@ function reverseIfRtl(chars) {
   return s;
 }
 
+function normalizeUnicode(str, normalizedUnicodesMap) {
+  const unicodeBuf = [];
+  for (let i = 0, ii = str.length; i < ii; i++) {
+    const unicode = str[i];
+    const normalized = normalizedUnicodesMap[unicode];
+    if (normalized === undefined) {
+      // If a toUnicode entry contains two same unified Unicode
```

Comment:
Let's replace `same` with `identical`, and please remember to end all sentences with a period.